examples/custom-config-ssl/configs/postgres-ha.yaml

```diff
@@ -0,0 +1,19 @@
+---
+bootstrap:
+  dcs:
+    postgresql:
+      parameters:
+        log_directory: pglogs
+        log_min_duration_statement: 0
+        log_statement: none
+        max_wal_senders: 6
+        shared_preload_libraries: pgaudit.so
+postgresql:  
+  pg_hba:
+    - local all postgres peer
+    - local all crunchyadm peer
+    - hostssl replication primaryuser 0.0.0.0/0 cert clientcert=1
+    - hostssl all primaryuser 0.0.0.0/0 reject
+    - hostssl all all 0.0.0.0/0 cert clientcert=1 map=ssl-testuser
```

Comment:
This might be accidentally blocking `pgo [create|update|delete] user` for this iteration, unless we modify those commands to use peer auth. For the latter (peer auth) method, what would hurt as at this moment is that we allow for selectors to do that in bulk and the command is synchronous, so it would be a very costly operation

Reply:
The following line was added in a subsequent commit: 7549466

```
- host all postgres 0.0.0.0/0 md5
```